openmls/src/group/core_group/test_proposals.rs

```diff
@@ -310,58 +309,6 @@ fn test_required_unsupported_proposals(ciphersuite: Ciphersuite, provider: &impl
     )
 }
 
-#[apply(ciphersuites_and_providers)]
```

Comment:
Instead of deleting this test, can't you replace the default extensions used here with extensions of type `Unknown`?

Reply:
I see. Then I think it's okay to delete it for now. I'll restore it once I get around to enabling Unknown extensions.